pkg/datapath/linux/config/config.go

```diff
@@ -20,8 +20,8 @@ import (
 
 	"github.com/cilium/cilium/pkg/bpf"
 	"github.com/cilium/cilium/pkg/byteorder"
-	"github.com/cilium/cilium/pkg/datapath"
 	"github.com/cilium/cilium/pkg/datapath/link"
+	"github.com/cilium/cilium/pkg/datapath/types"
```

Comment:
Why not import this as `datapath "github.com/cilium/cilium/pkg/datapath/types"` everywhere to avoid the code churn? Callers are only going to want types in 99% of cases, so it makes sense to keep referring to them as `datapath.LocalNodeConfiguration` even after them being moved.

Also makes backports painless, is going to get tedious otherwise.

Reply:
whoof, why didn't I think of this :face_exhaling: will do, thanks for the idea!

Reply:
diff much smaller now :100: